scripts/utils/metadata_interaction.py

```diff
@@ -161,7 +161,10 @@ def find_type_id_in_metadata(name, metadata_types, default_path_marker='typeName
                     else:
                         value = value[path]
                         continue
-                return value['type']
+                try:
```

Comment:
I would prefer this approach, so we avoid processing the exact exception:
```suggestion
                elif value.get('type') is None:
                    return value['index']
                else:
                    return value['type']
```